WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/CollapsibleShipmentCard.swift

```diff
@@ -0,0 +1,72 @@
+import Yosemite
+import SwiftUI
+
+/// Displays a single collapsible shipment item row or grouped parent and child shipment item rows
+struct CollapsibleShipmentCard: View {
+    @State private var isCollapsed: Bool = true
+
+    private let viewModel: CollapsibleShipmentCardViewModel
+
+    init(viewModel: CollapsibleShipmentCardViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            mainShipmentRow
+                .padding(.horizontal, Layout.horizontalPadding)
+                .padding(.vertical, Layout.verticalPadding)
+                .background(Color(.listForeground(modal: false)))
```

Comment:
The view has a rounded border so the background view should be rounded as well to avoid showing sharp corners underneath (more visible in dark mode).

```suggestion
                .background(
                     RoundedRectangle(cornerRadius: Layout.borderCornerRadius)
                         .fill(Color(.listForeground(modal: false)))
                )
```

Reply:
Good eye for design. 🕵️ 

I have updated the logic to apply corner radius based on current UI state in https://github.com/woocommerce/woocommerce-ios/pull/15397/commits/2c64d09fdd17e4ecb0d83076bc0e3ba69e2fa237 and https://github.com/woocommerce/woocommerce-ios/pull/15397/commits/36a64c6bb80321ac74081dd38579308bc6f29da6. 

| Before | After |
|--------|--------|
| ![Before](https://github.com/user-attachments/assets/1127f75f-6f04-4eea-b2ee-5ca07ab40d8e) | ![After](https://github.com/user-attachments/assets/b48c659e-fa01-4e60-a8f3-96c60cf37ac5) | 

The bleeding corners should be fixed now for both main and child shipment rows.